spec/views/shared/avis/list.html.haml_spec.rb

```diff
@@ -1,7 +1,7 @@
-describe 'instructeurs/avis/_list.html.haml', type: :view do
+describe 'shared/avis/_list.html.haml', type: :view do
   before { view.extend DossierHelper }
 
-  subject { render 'instructeurs/avis/list.html.haml', avis: avis, avis_seen_at: seen_at, current_instructeur: instructeur }
+  subject { render 'shared/avis/list.html.haml', avis: avis, avis_seen_at: seen_at, current_instructeur: instructeur, expert_or_instructeur: instructeur }
```

Comment:
il me semble qu'on peut virer `current_instructeur`

```suggestion
  subject { render 'shared/avis/list.html.haml', avis: avis, avis_seen_at: seen_at, expert_or_instructeur: instructeur }
```